src/bundle/Controller/PasswordResetController.php

```diff
@@ -74,7 +78,8 @@ public function __construct(
         NotificationHandlerInterface $notificationHandler,
         PermissionResolver $permissionResolver,
         string $tokenIntervalSpec,
-        string $forgotPasswordMail
+        string $forgotPasswordMail,
+        string $senderAddress
```

Comment:
```suggestion
        string $senderAddress = ''
```
Let's avoid BC breaks.